packages/cli/docsgen/markdown.ts

```diff
@@ -198,7 +202,7 @@ function renderSubCommand(sub: SubCommandDefinition, lodestarCommand?: string):
       renderOptions(
         sub.options,
         `### \`${sub.command}\` Options`,
-        "_Supports all parent command options plus the following:_\n\n<br>"
+        "_Supports all parent command options plus the following:_\n\n"
```

Comment:
```suggestion
        `_Supports all parent command options plus the following:_${LINE_BREAK}`
```

Reply:
Is that a docusaurus thing? Couldn't find mention of it.

Reply:
No. That was a remnant from updating the script to build the cli docs.  In some places the line separator needed the extra `<br />` but in other it did not to look correct.  As long as it no longer a thing we can just use the single LINE_BREAK const.